src/Node.ipp

```diff
@@ -102,6 +135,36 @@ ServiceServer Node::create_service_server(CanardPortID const request_port_id, Ca
     *this,
     request_port_id,
     tx_timeout_usec,
+    CanardPriorityNominal,
```

Comment:
Same as for create_publisher, this is a lot of code duplication now. Replace with

```C++
return create_service_server<T_REQ, T_RSP, OnRequestCb>(request_port_id, tx_timeout_usec, on_request_cb, CanardPriorityNominal. tid_timeout_usec); // Probably needs std::forward for on_request_cb
```